src/llm/llmnoderesources.cpp

```diff
@@ -166,6 +166,9 @@ Status LLMNodeResources::createLLMNodeResources(std::shared_ptr<LLMNodeResources
 
     nodeResources->initiateGeneration();
 
+    nodeResources->maxTokensLimit = nodeOptions.max_tokens_limit();
+    nodeResources->bestOfLimit = nodeOptions.best_of_limit();
```

Comment:
Please move this above `initiateGeneration()` call. 
Also can we create some structure or map with `requestLimits` instead of separate fields? It would be easier to extend in the future. Then you could pass `nodeResource->requestLimits` to `parse` method.